specs/windows/disk_info.table

```diff
@@ -0,0 +1,16 @@
+table_name("disk_info")
+description("Retrieve basic information about the physical disks of a system.")
+schema([
+    Column("partitions", INTEGER, "Number of detected partitions on disk."),
+    Column("disk_index", INTEGER, "Physical drive number of the disk."),
```

Comment:
Maybe just call this `index`, as the disk seems redundant.

Reply:
I did originally use index, but on compiling it threw an error because it saw index as something else. I'm going to guess that's a reserved name in that context

Reply:
Good call, stick with `disk_index`.